website/docs/d/wafregional_web_acl.html

```diff
@@ -0,0 +1,30 @@
+---
+layout: "aws"
+page_title: "AWS: aws_wafregional_web_acl"
+sidebar_current: "docs-aws-datasource-wafregional-web-acl"
+description: |-
+Retrieves a WAF Regional Web ACL id.
+---
+
+# Data Source: aws_wafregional_web_acl
+
+`aws_wafregional_web_acl` Retrieves a WAF Regional Web ACL Resource Id.
+
+## Example Usage
+
+```hcl
+data "aws_wafregional_web_acl" "example" {
+name = "tfWAFRule"
```

Comment:
Terraform formatting:

```suggestion
  name = "tfWAFRule"
```